### ISSUE TITLE
Issues using node 9.3.0

### ISSUE THREAD

**Issue by ewanharris** [abbr/deasync#89]
Hey there,

When using deasync on node 9.3.0 it errors with the below (I'm just running the quick-test.js script)
````
C:\Users\ewan\Documents\git\deasync\index.js:71
                process._tickDomainCallback();
                        ^

TypeError: process._tickDomainCallback is not a function
    at Function.module.exports.loopWhile (C:\Users\ewan\Documents\git\deasync\index.js:71:11)
    at Function.sleep (C:\Users\ewan\Documents\git\deasync\index.js:44:19)
    at Object.<anonymous> (C:\Users\ewan\Documents\git\deasync\quick-test.js:6:34)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Function.Module.runMain (module.js:701:10)
    at startup (bootstrap_node.js:194:16)
````

fwiw I tested  #75 and this looks to fix the issue, but is still using the removed api when process.domain is true, and _tickCallback is internal/undocumented also

**Comment by jaikdean** [comment]
I'm seeing the same issue.

**Comment by abbr** [comment]
Should be fixed by deasync@0.1.12.
